utils/logger_config.py

```
@@ -147,8 +148,8 @@ def _get_log_directory(self):
             print(f"Warning: Failed to use temp directory: {e}", file=sys.stderr)
         
         # 如果所有方法都失败，返回当前目录
-        print(f"Warning: All log directory attempts failed, using current directory", file=sys.stderr)
-        return Path.cwd() / "logs"
+        print(f"Warning: All log directory attempts failed, using application directory", file=sys.stderr)
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)